<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Welcome to Tic-Tac-Toe !</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>730</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QStackedWidget" name="stacked">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="configPage">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QFrame" name="frame">
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>400</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>15</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Difficulty</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QSlider" name="horizontalSlider">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>15</height>
                     </size>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="difficulty">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>15</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>1</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_13">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>15</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>%</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>15</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Play</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QRadioButton" name="playFirst">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>15</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>First</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup_2</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="playSecond">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>15</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Second</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup_2</string>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QPushButton" name="start">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="gamePage" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="4">
                 <widget class="Line" name="line_10">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="Line" name="line_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="Line" name="line_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="Line" name="line_6">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item row="4" column="3">
                 <widget class="Line" name="line_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="Line" name="line_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="Line" name="line_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="Line" name="line_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="Line" name="line">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="Line" name="line_11">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="4">
                 <widget class="Line" name="line_9">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="Line" name="line_12">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="map1">
                  <property name="font">
                   <font>
                    <pointsize>64</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="map2">
                  <property name="font">
                   <font>
                    <pointsize>64</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="4">
                 <widget class="QPushButton" name="map3">
                  <property name="font">
                   <font>
                    <pointsize>64</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QPushButton" name="map5">
                  <property name="font">
                   <font>
                    <pointsize>64</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="4">
                 <widget class="QPushButton" name="map6">
                  <property name="font">
                   <font>
                    <pointsize>64</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="4">
                 <widget class="QPushButton" name="map9">
                  <property name="font">
                   <font>
                    <pointsize>64</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="2">
                 <widget class="QPushButton" name="map8">
                  <property name="font">
                   <font>
                    <pointsize>64</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="map4">
                  <property name="font">
                   <font>
                    <pointsize>64</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QPushButton" name="map7">
                  <property name="font">
                   <font>
                    <pointsize>64</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="restartButton">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Restart</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
